Source/diablo.cpp

```diff
@@ -437,6 +437,7 @@ static void SaveOptions()
 	setIniInt("Game", "Auto Equip Shields", sgOptions.Gameplay.bAutoEquipShields);
 	setIniInt("Game", "Auto Equip Jewelry", sgOptions.Gameplay.bAutoEquipJewelry);
 	setIniInt("Game", "Enable All Quests for Single Player", sgOptions.Gameplay.bAllQuests);
+	setIniInt("Game", "Show monster type information", sgOptions.Gameplay.bShowMonsterType);

```

Comment:
Is there other information other than the type itself?
If not, consider renaming to "Show monster type"

Reply:
Call it "Show Monster Type" so the casing is also consistent.

Reply:
Renamed. And fixed casing to match other setting names.